Src/GraphicLibs/Ncurses/NcursesLibrary.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+** EPITECH PROJECT, 2022
+** arcAAAAAAAAAAAAAAAAade
+** File description:
+** NcursesLibrary
+*/
+
+#include "NcursesLibrary.hpp"
+
+NcursesLibrary::NcursesLibrary()
+{
+    initscr();
+}
+
+NcursesLibrary::~NcursesLibrary()
+{
+    endwin();
+}
+
+bool NcursesLibrary::clearWindow()
+{
+    clear();
+    refresh();
+    return true;
+}
+
+bool NcursesLibrary::displayWindow()
+{
+    return true;
+}
+
+bool NcursesLibrary::drawCircle(std::pair<int, int> pos, int radius, Color color)
+{
```

Comment:
drawCircle need to do something even if it's the same as rect because the core didn't know wich graphic library u are using, so if u print circle with a random library, u should be able to print a circle on ncurse (even if it's a rectangle)